streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -364,5 +364,15 @@ private AssignmentConfigs(final StreamsConfig configs) {
             this.numStandbyReplicas = numStandbyReplicas;
             this.probingRebalanceIntervalMs = probingRebalanceIntervalMs;
         }
+
+        @Override
+        public String toString() {
+            return "AssignmentConfigs{" +
+                "\n  acceptableRecoveryLag=" + acceptableRecoveryLag +
+                "\n  maxWarmupReplicas=" + maxWarmupReplicas +
+                "\n  numStandbyReplicas=" + numStandbyReplicas +
+                "\n  probingRebalanceIntervalMs=" + probingRebalanceIntervalMs +
+                "\n}";
```

Comment:
prop: Wouldn't it be better to use the config names as labels instead of the names of the member fields?
```suggestion
            return "AssignmentConfigs{" +
                "\n " + StreamsConfig.ACCEPTABLE_RECOVERY_LAG_CONFIG + " = " + acceptableRecoveryLag +
                "\n " + StreamsConfig.MAX_WARMUP_REPLICAS_CONFIG + " = " + maxWarmupReplicas +
                "\n " + StreamsConfig.NUM_STANDBY_REPLICAS_CONFIG + " = " + numStandbyReplicas +
                "\n " + StreamsConfig.PROBING_REBALANCE_INTERVAL_MS_CONFIG + " = " + probingRebalanceIntervalMs +
                "\n}";
```

Reply:
Huh, thanks for the suggestion!

I guess I'm weakly opposed. It seems strange for the toString representation of an object not to represent the field of that object. Not for any good reason, except that that's what I typically see. It's not for "public" consumption in any way; it just gets printed in our own unit tests if they fail. In that case, it actually seems more roundabout because we don't use the config property names in those unit tests, we directly instantiate the AssignmentConfig object, which uses the same field names as constructor arguments.

It's an interesting idea, but if it's ok with you, I'll just leave it as "vanilla" (autogenerated) style for now.

Reply:
Fair enough